Gordon360/Models/CCT/Context/efpt.CCT.config.json

```diff
@@ -8,6 +8,7 @@
    "ModelNamespace": null,
    "OutputContextPath": "Models\/CCT\/Context",
    "OutputPath": "Models\/CCT",
+   "PreserveCasingWithRegex": true,
```

Comment:
@EjPlatzer looks like this appeared in their version as well (you made a comment a few weeks ago about it in my PR). Likely just a updated default on EFCore, so I think we can let it pass.